data/schema.graphql

```diff
@@ -1264,6 +1287,15 @@ enum BuyerOutcomeTypes {
 }
 
 type BuyersPremium {
+  # A globally unique ID.
+  __id: ID!
+
+  # A type-specific ID.
+  id: String!
+
+  # A type-specific Gravity Mongo Document ID.
+  _id: String!
+
```

Comment:
It’s the `id` field that was already in use by us in a non-globally unique way, and as `_id` also already existed we arrived at `__id`.

For more info see https://facebook.github.io/relay/graphql/objectidentification.htm